src/main/java/stsjorbsmod/powers/OldBookPower.java

```diff
@@ -0,0 +1,65 @@
+package stsjorbsmod.powers;
+
+import com.megacrit.cardcrawl.actions.common.RemoveSpecificPowerAction;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.AbstractCreature;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.powers.AbstractPower;
+import stsjorbsmod.actions.ExhumeCardsAction;
+import stsjorbsmod.util.CardMetaUtils;
+
+public class OldBookPower extends CustomJorbsModPower {
+    public static final StaticPowerInfo STATIC = StaticPowerInfo.Load(OldBookPower.class);
+    public static final String POWER_ID = STATIC.ID;
+
+    private AbstractCard card;
+
+    public OldBookPower(final AbstractCreature owner, AbstractCard card) {
+        super(STATIC);
+
+        this.isTurnBased = false;
+        this.card = card;
+        this.owner = owner;
+
+        updateDescription();
+    }
+
+    @Override
+    public int onAttacked(DamageInfo info, int damageAmount) {
+        if (damageAmount >= owner.currentHealth) {
+            info.output = damageAmount = 0;
+            addToBot(new ExhumeCardsAction(card));
```

Comment:
I don't think this is actually going to do anything; `destroyCardPermanently` is going to *immediately* remove the card from the exhaust pile, but this action is only going to search the exhaust pile for the card to exhume later, once the action is pulled off the queue.